recipe-box-frontend/src/main/frontend/src/components/IngredientList/UpdateIngredientQuantityPopUp.tsx

```diff
@@ -0,0 +1,68 @@
+import { ChangeEvent } from "react";
+import { ReactComponent as CloseButton } from "../../assets/icons/close.svg";
+import "./UpdateIngredientQuantityPopUp.css";
+interface UpdateIngredientQuantityPopUpProps {
+    setPopUpIngredientQuantity: React.Dispatch<React.SetStateAction<number>>;
+    popUpIngredientQuantity: number;
+    setAddPopUpToggled: React.Dispatch<React.SetStateAction<boolean>>;
+    ingredient: { id: number; unit: string };
+}
+
+export function UpdateIngredientQuantityPopup({
+    setPopUpIngredientQuantity,
+    popUpIngredientQuantity,
```

Comment:
these two props at line 12 and 13 could be just value and setValue, no need for long name props